### PR TITLE
feat(amazon/instance): Switch General Purpose instance from m4 -> m5

### PR DESCRIPTION
**Pull request by erikmunson**
Now that m5 instances are battle tested, this changes our recommended family for normal/average uses from m4 -> m5. I also tweaked the language in the description, because it was phrased kind of strangely.

### CODE REVIEW

File: app/scripts/modules/amazon/src/instance/awsInstanceType.service.js [hunk 1]
```diff
@@ -8,34 +8,34 @@ import { API } from '@spinnaker/core';
 module.exports = angular
   .module('spinnaker.amazon.instanceType.service', [])
   .factory('awsInstanceTypeService', function($http, $q) {
-    var m4 = {
-      type: 'm4',
+    var m5 = {
+      type: 'm5',
       description:
-        'This family includes the m4 instance types and provides a balance of compute, memory, and network resources, and it is a good choice for many applications.',
+        'm5 instances provide a balance of compute, memory, and network resources. They are a good choice for most applications.',
```
Comment (erikmunson) [inline app/scripts/modules/amazon/src/instance/awsInstanceType.service.js]: changed from `many` to `most` applications as a result of some feedback/learning we've gotten internally.

**Comment by jrsquared** [comment]
`because it was phrased kind of strangely` says the person who ends a sentence with an adverb.

**Comment by jrsquared** [comment]
I like it. LGTM.

File: app/scripts/modules/amazon/src/instance/awsInstanceType.service.js [hunk 2]
```diff
@@ -8,34 +8,34 @@ import { API } from '@spinnaker/core';
...
         },
         {
-          name: 'm4.2xlarge',
+          name: 'm5.2xlarge',
           label: '2XLarge',
           cpu: 8,
           memory: 32,
           storage: { type: 'EBS' },
-          costFactor: 2,
+          costFactor: 4,
```
Comment (erikmunson) [inline app/scripts/modules/amazon/src/instance/awsInstanceType.service.js]: Upped this because technically a 2xl is double the cost of an xl.

**Comment by erikmunson** [comment]
> `because it was phrased kind of strangely` says the person who ends a sentence with an adverb.

I wanted to put focus on the adverb, Justin. Thanks for reading so _diligently_.

**Pull request merged**